Analysis/include/DYModel.h

```diff
@@ -60,7 +62,7 @@ class DYModel {
         }
         if(fit_method == DYFitModel::NbjetBins || fit_method == DYFitModel::NbjetBins_htBins ||
                 fit_method == DYFitModel::NbjetBins_NjetBins){
-            auto input_file = root_ext::OpenRootFile(sample.norm_sf_file);
+            auto input_file = root_ext::OpenRootFile(working_path+sample.norm_sf_file);
```

Comment:
working_path + "/" + sample.norm_sf_file
(in case if someone don't put / at the end)